1.9/usage/pods/using-pods.md

```diff
@@ -0,0 +1,23 @@
+---
+post_title: Using Pods
+menu_order: 10
+---
+
+DC/OS treats and represents pods as single services. From the perspective of Marathon, pods are pods are members of [groups](http://mesosphere.github.io/marathon/docs/application-groups.html). Containers in pods share networking namespace and ephemeral volumes.
+
+You can create and manage pods with the [DC/OS CLI](/docs/1.9/usage/pods/pods-cli/) or via the /v2/pods endpoint of the [Marathon REST API](http://mesosphere.github.io/marathon/docs/generated/api.html).
+
+# Pod Definitions
+Pods are configured via a JSON pod definition, which is similar to a Marathon application definition. [link to docs on that]. See the [Examples](/docs/1.9/usage/pods/examples/) section for more information. You must declare the resources required by each container in the pod, even if Mesos is isolating at a higher (pod) level. 
+
+**Note:** DC/OS reserves 32MB and .1 CPUs per pod for overhead. Take this overhead into account when declaring resource needs for the containers in your pod.
+
+# Networking
+Pods only support the Mesos containerizer, which simplifies networking among containers. The containers of each pod instance share a network namespace and can communicate over localhost. 
+
+Containers in pods are created with endpoints. Other applications communicate with pods by addressing those endpoints. A default container network may be specified on the Marathon command line. If you specify a container network without a name in a pod definition, it will be assigned to this default network.
```

Comment:
Is `Marathon command line` the same as the `dcos marathon` command? Suggest rewrite to `You can specify a default container network by using the CLI dcos marathon command.`